building/build-debs/homeworld-keysystem/src/keytool.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
What is this here for?  In fact, why do we have 4 incompatible definitions of a single `main` package at the top level?  Shouldn't e.g. `keyserver.go` be in a subdirectory of `keyserver`, etc?

Reply:
These are in `main` because they need to be compiled into binaries? I'm not fully clear on how this system worked, but I can't just make these a subpackage of, say, keyserver, as far as I can tell. Go will just not produce a binary.

Reply:
You can put `keyserver.go` into `keyserver/main/`.

Reply:
Ahh, alright.